### DISCUSSION TITLE
Use more env variables to configure docker mealie

### DISCUSSION THREAD

**Discussion by floreo** [mealie-recipes/mealie#935] [Feature Request]
**Before submitting this feature request I have**

- [X] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X] Reviewed the existing feature requests and my feature has not been requested.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

Hello,

it would be practical to have some more ENV variables to configure mealie on docker:

* **LANG**: define default language of mealie (en_US.UTF-8, fr_FR.UTF-8, ...)
* **FIRST_WEEK_DAY**: define first day of the week (monday, sunday, ...)

**LANG** is already available, but setting it wouldn't change the default language of mealie.

Thanks.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
First day of the week is currently customizable in group ([soon to be household](https://github.com/mealie-recipes/mealie/pull/3970)) settings. Currently there's no way to set the default language AFAIK
